src/components/IstioWizards/WizardActions.ts

```diff
@@ -1137,6 +1137,87 @@ export const buildAuthorizationPolicy = (
   return ap;
 };
 
+export const buildGraphSidecars = (namespace: string, graph: GraphDefinition): Sidecar[] => {
+  const denyAll: Sidecar = {
+    metadata: {
+      name: 'deny-all-' + namespace,
+      namespace: namespace,
+      labels: {
+        [KIALI_WIZARD_LABEL]: 'Sidecar'
+      }
+    },
+    spec: {
+      egress: [
+        {
+          hosts: ['./*', 'istio-system/*']
+        }
+      ]
+    }
+  };
+
+  const sidecars: Sidecar[] = [denyAll];
+
+  if (graph.elements.nodes) {
+    for (let i = 0; i < graph.elements.nodes.length; i++) {
+      const node = graph.elements.nodes[i];
+      if (
+        node.data.namespace === namespace &&
+        node.data.nodeType === NodeType.WORKLOAD &&
+        node.data.workload &&
+        node.data.app &&
+        node.data.version
+      ) {
+        const sc: Sidecar = {
+          metadata: {
+            name: node.data.workload,
+            namespace: namespace,
+            labels: {
+              [KIALI_WIZARD_LABEL]: 'AuthorizationPolicy'
+            }
+          },
+          spec: {
+            workloadSelector: {
+              labels: {
+                app: node.data.app,
+                version: node.data.version
+              }
+            },
+            egress: [
+              {
+                hosts: ['istio-system/*']
+              }
+            ]
+          }
+        };
+
+        if (graph.elements.edges) {
+          for (let j = 0; j < graph.elements.edges.length; j++) {
+            const edge = graph.elements.edges[j];
+
+            if (node.data.id === edge.data.source) {
+              for (let z = 0; z < graph.elements.nodes.length; z++) {
+                const targetNode = graph.elements.nodes[z];
+
+                if (targetNode.data.id === edge.data.target) {
+                  targetNode.data.destServices?.forEach((ds: DestService) => {
+                    if (sc.spec.egress && ds.namespace !== 'unknown') {
+                      sc.spec.egress[0].hosts.push(`${ds.namespace}/${ds.name}.${ds.namespace}.svc.cluster.local`);
```

Comment:
I'd try to fetch the constants like 'svc.cluster.local' from the config (there is a global 'serverConfig'):

![image](https://user-images.githubusercontent.com/1662329/135598092-1723858f-c77b-4931-a108-86fdb045f2bb.png)

Then if those values change we are safe.